chsdi/models/__init__.py

```diff
@@ -65,7 +65,8 @@ def set_models_srid(models, srid):
         return models
     ms = []
     for m in models:
-        m.set_geometry_srid_out(srid)
+        # TODO remove completely
+        # m.set_geometry_srid_out(srid)
```

Comment:
maybe the class method should be removed as well (or at least commented)?

Reply:
OK, done.